src/config/configmanager.cpp

```diff
@@ -127,6 +127,7 @@ bool ConfigManager::load() {
 	loadBoolConfig(L, REMOVE_WEAPON_CHARGES, "removeWeaponCharges", true);
 	loadBoolConfig(L, REPLACE_KICK_ON_LOGIN, "replaceKickOnLogin", true);
 	loadBoolConfig(L, REWARD_CHEST_COLLECT_ENABLED, "rewardChestCollectEnabled", true);
+	loadBoolConfig(L, SHOW_LOOTS_IN_BESTIARY, "showLootsInBestiary", false);
```

Comment:
please, put this config after `SCRIPTS_CONSOLE_LOGS` (alphabetical order)

Reply:
done.